src/commands/groq.rs

```diff
@@ -36,7 +36,7 @@ impl CommandTrait for Llama {
             ctx.bot_state.http_client.clone(),
             "https://api.groq.com/openai/v1",
             &env::var("GROQ_API_KEY").unwrap(),
-            "llama3-70b-8192",
+            "llama-3.1-70b-versatile",
```

Comment:
can we update to `llama-3.1-405b-reasoning` or is the demand too high?

Reply:
few reasons why this is a bad idea
- it's slow
- absurd ratelimits
- very high demand
- wayyyyyy overkill
- just why?

Reply:
> it's slow
absurd ratelimits
very high demand

this answers my question

> wayyyyyy overkill
just why?

because i'm yet to see a model that doesn't feel like talking with a 5-year-old